package.json

```diff
@@ -27,6 +27,12 @@
         "type": "git",
         "url": "git+https://github.com/supertokens/supertokens-node.git"
     },
+    "exports": {
+        "./recipe/*": {
+            "types": "./recipe/*/index.d.ts",
+            "default": "./recipe/*/index.js"
```

Comment:
Were you able to test this with both ESM and CJS node applications? I'd be surprised if we didn't need distinct export paths for CJS/ESM

Reply:
Yeah in the test video you can see in the test-js side I have tried importing while having type module and without it as well 
Using require and import both